script/run-unit-test.sh

```diff
@@ -11,8 +11,12 @@ fi
 # No test specified; run all of them
 if [ -z "$1" ]; then
   BABEL_ENV=test $MOCHA --recursive '$SCRIPT_DIR/../test/**/*.unit.spec.js?(x)' 
-  exit 0
+  exit $?
```

Comment:
Oh bash. Why you gotta be like this? Of _course_  `$? ` means `the exit status of the previously executed command`.

:man_shrugging: